mmv1/third_party/terraform/services/memcache/data_source_memcache_instance_test.go

```diff
@@ -0,0 +1,65 @@
+package memcache_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccMemcacheInstanceDatasourceConfig(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckMemcacheInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMemcacheInstanceDatasourceConfig(context),
```

Comment:
Could you add `acctest.CheckDataSourceStateMatchesResourceStateWithIgnores` here? Example: https://github.com/hashicorp/terraform-provider-google-beta/blob/ce9cdb5e614479e0fd65dddb932d09c775d6373e/google-beta/services/sql/data_source_sql_database_test.go#L27 (The compose wrapper probably isn't necessary).

Reply:
You bet I can